### ISSUE TITLE
Bug: ERROR creating tun device: unix opening TUN device file: operation not permitted

### ISSUE THREAD

**Issue by sammyt107** [qdm12/gluetun#2626]
### Is this urgent?

Yes

### Host OS

Debian OMV

### CPU arch

x86_64

### VPN service provider

Private Internet Access

### What are you using to run the container

Portainer

### What is the version of Gluetun

latest built on 	2024-12-14 21:09:55

### What's the problem 🤔

I'm getting this creating tun error on portainer, tried using latest and v3 versions but doesn't work on either.  

### Share your logs (at least 10 lines)

```plain text
2024-12-22T20:28:21Z ERROR creating tun device: unix opening TUN device file: operation not permitted
```

**Comment by GentleHoneyLover** [comment]
I have the same error since the last container image update...

**Comment by sammyt107** [comment]
@GentleHoneyLover Thanks for the help and that seems to have worked, will leave open for now to see if we get a response on the tun device mapping

**Comment by danthonywalker** [comment]
I am getting the same issue. I haven't changed versions, all I have done recently is restart my system and suddenly this issue is appearing. The workaround from @GentleHoneyLover works, but I'm wondering what has changed.

**Comment by raph521** [comment]
containerd.io was likely updated on your system.

There's another issue about it with more details:
https://github.com/qdm12/gluetun/issues/2606

**Comment by qdm12** [comment]
Duplicated of now resolved #2606

About to cut a release which will log out a fix when encountering this error.
